frontend/coprs_frontend/coprs/logic/coprs_logic.py

```diff
@@ -691,6 +685,26 @@ def get_or_create(cls, copr, dirname):
                 f"Wrong directory '{dirname}' specified.  Directory name can "
                 "consist of alpha-numeric strings separated by colons.")
 
+    @classmethod
+    def get_or_validate(cls, copr, dirname):
```

Comment:
`get_or_validate` is a quite unusual concept. Do we need it? 
Can we do only `validate` in `apiv3_builds.py` and move the "get" part into `get_or_create`?

Reply:
We can do that, but wouldn't that be duplication?
I agree that's not usual :-/ though that's exactly the task we do on two places.

Reply:
I updated the doc-string to better explain why we are returning existing directories even if they do not match the current validation patterns.